application/logging_default_config.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
```

Comment:
Shall we rename this file as `log_config.yaml`?

There have been discussions about whether extension `.yml` should be used in favour of `.yaml`. Let's use `.yaml` according to this: https://www.rfc-editor.org/rfc/rfc9512.html#:~:text=The%20%22yaml%22%20filename%20extension%20is%20the%20preferred%20one

Reply:
I am thinking, would we have multiple config files or 1 for now is enough.

Reply:
What benefits do you perceive in maintaining multiple configurations?

Reply:
I am trying to think of possibilities, for now we are trying to stick to a consistent format and using the root logger by using getLogger so I guess one config file is enough.